model/jpa/src/main/java/org/keycloak/models/jpa/entities/RealmLocalizationTextsEntity.java

```diff
@@ -76,15 +77,25 @@ public int hashCode() {
     private String locale;
 
     @Column(name = "TEXTS")
-    @Convert(converter = MapStringConverter.class)
```

Comment:
I think we should either have a constant for `MapStringConverter` or just move its methods to this entity. The latter is probably better.

Reply:
I've updated my PR to use the constant. Also I've added TODO to revert this change once quarkus fix is applied and we can update to the quarkus version with the fix. I did not changed the "MapStringConverter" class itself. Is it fine for you that way?